pkg/skaffold/docker/client.go

```diff
@@ -173,10 +173,9 @@ func newMinikubeAPIClient(minikubeProfile string) ([]string, client.CommonAPICli
 }
 
 func getUserAgentHeader() map[string]string {
-	userAgent := fmt.Sprintf("skaffold-%s", version.Get().Version)
-	logrus.Debugf("setting Docker user agent to %s", userAgent)
+	logrus.Debugf("setting Docker user agent to %s", version.UserAgent())
 	return map[string]string{
-		"User-Agent": userAgent,
+		"User-Agent": version.UserAgent(),
```

Comment:
why don't we want to add client info here as well?

Reply:
For two reasons
1) This metric tracks docker pushes to GCR built by skaffold
2) This will break the logic we have internally to detect if the image pushed by built by skaffold.